.yamato/hdrp_performancetests-win-dx11.yml

```diff
@@ -18,21 +18,19 @@ HDRP_PerformanceTests_Win_DX11_performance_memory_il2cpp_Linear_trunk:
         UTR_VERSION: "current"
         TEST_FILTER: .*
     commands:
-      -  curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output TestProjects/HDRP_PerformanceTests/utr.bat
       -  choco install unity-downloader-cli -y -s https://artifactory.prd.it.unity3d.com/artifactory/api/nuget/unity-choco-local
+      -  unity-downloader-cli -u {{editor_versions.trunk_latest_internal.Win.revision}} -c editor -c il2cpp  --wait --published-only
       -  NetSh Advfirewall set allprofiles state off
-      -  cd TestProjects/HDRP_PerformanceTests && unity-downloader-cli -u {{editor_versions.trunk_latest_internal.windows.revision}} -c editor -c il2cpp  --wait --published-only
-      -  |5-
-
-                  cd TestProjects/HDRP_PerformanceTests && utr --artifacts_path=test-results --editor-location=.Editor --extra-editor-arg="-colorspace=Linear" --extra-editor-arg="-force-d3d11" --loglevel=verbose --performance-project-version=../.. --report-performance-data --performance-project-id=HDRP --platform=StandaloneWindows64 --reruncount=2 --scripting-backend=IL2CPP --suite=playmode --testfilter=Memory --testproject=. --timeout=2400 --zero-tests-are-ok=1
+      -  curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr.bat --output utr.bat
+      -  utr --artifacts_path=TestProjects/HDRP_PerformanceTests/test-results --editor-location=.Editor --extra-editor-arg="-colorspace=Linear" --extra-editor-arg="-force-d3d11" --loglevel=verbose --performance-project-version=../.. --report-performance-data --performance-project-id=HDRP --platform=StandaloneWindows64 --reruncount=2 --scripting-backend=IL2CPP --suite=playmode --testfilter=Memory --testproject=./TestProjects/HDRP_PerformanceTests --timeout=2400 --zero-tests-are-ok=1
     artifacts:
         logs:
             paths:
               -  "**/test-results/**"
               -  "TestProjects/HDRP_PerformanceTests/Logs/*.log"
               -  "PrebuildLog/**"
 HDRP_PerformanceTests_Win_DX11_performance_run_build_il2cpp_Linear_trunk:
-    name: Build HDRP_PerformanceTests on Win_DX11_il2cpp_Linear_performance_run_build_Player on version trunk
+    name: HDRP_PerformanceTests on Win_DX11_performance_run_build_il2cpp_Linear on version trunk
```

Comment:
theo knows more about the hdrp perf jobs than me but i dont think this should have changed

Reply:
good point, @theo-pnv can you check this? 
I added this intentionally for now. All the other jobs which's name starts with 'Build' are dependencies, but this job seems not to be a dependency, but an 'unit' on it's own. Because the job_id seemed to be one of a normal job (not a build dependency), this previous job name (looking like build dependency) seemed by accident. If that's not the case, I'll reverse this change!

Reply:
Ah, let's keep the new version, you got it correctly @kivistikliis! I should have renamed it properly when making the perf PR.
This is indeed a test on its own, not a dependency. From what Antoine told me this test is supposed to build the editor and get info about the shader compil time, build sizes, etc. It doesn't work anyway, so it's not in any nightly yet.
So the new version is correct. Thanks for catching this one and raising awareness @sophiaaar .